### DISCUSSION TITLE
Is it permissible to have installation twice of MSYS2?

### DISCUSSION THREAD

**Discussion by ghost** [msys2/MSYS2-packages#4846] [Q&A] [answered]
I want to use them with different propose. One of them is just coding and compiling, and the other is compile other software and install them without `pacman`.

**Comment by Biswa96** [comment, marked as answer]
You can absolutely do that. Though I am not sure why you want to do that.

**Reply by sskras** [reply]
Eg. to have two sets of software: one set installed via `pacman`, another set built from source. Or maybe to test installing software using different package manager (such as `apk`, `pkgsrc`, `nix` or else).

**Reply by Biswa96** [reply]
> another set built from source

It can be used from build directory.

> Or maybe to test installing software using different package manager (such as `apk`, `pkgsrc`, `nix` or else).

We are not talking about Linux or BSD.

**Reply by ghost** [reply]
> Eg. to have two sets of software: one set installed via `pacman`, another set built from source. Or maybe to test installing software using different package manager (such as `apk`, `pkgsrc`, `nix` or else).

Yes, that's what I want to say. I'm satisified to install some library that not appeared on MSYS2. :D

**Reply by sskras** [reply]
> > Or maybe to test installing software using different package manager (such as `apk`, `pkgsrc`, `nix` or else).
> 
> We are not talking about Linux or BSD.

@Biswa96, `pacman` also came from Linux. I think nothing prevents a person to try building a cross-platform software on one MSYS2 setup, and test it on another MSYS2 setup :)
